### PR TITLE
Add new product(s): products/project-connect.json

### PR DESCRIPTION
**Pull request by lacabra**
Add new product(s) from [unicef/publicgoods-candidates](https://github.com/unicef/publicgoods/candidates)

### CODE REVIEW

File: products/project-connect.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      "SDGNumber": 10,
+      "evidenceText": "Project Connect continues as a venture inside UNICEF's Office of Innovation within Magic Box, a platform for partners to come and connect their datasets and expertise to better understand the situation of children and youth, and to create insights that will help governments and other humanitarian actors improve their programmes around education, health and emergencies.",
+      "evidenceURL": "https://www.projectconnect.world/"
+    }
+  ],
+  "sectors": [],
+  "type": [
+    "software"
+  ],
+  "repositoryURL": "https://github.com/unicef/magicbox-maps",
```
Comment (conradsp) [inline products/project-connect.json:32]: It says that this repository has been archived. Is this project still active?

File: products/project-connect.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Project Connect",
+  "description": "Project Connect aims to create a global, live map of schools and their connection to the internet",
+  "website": "https://www.projectconnect.world/",
+  "license": [
+    {
+      "spdx": "BSD-3-Clause",
+      "licenseURL": "https://github.com/unicef/magicbox-maps"
```
Comment (conradsp) [inline products/project-connect.json:8]: There is no license file in the repository.

**Review by conradsp — CHANGES_REQUESTED** [review]

File: products/project-connect.json (hunk 1 above)
Comment (lacabra) [reply products/project-connect.json:32]: Yes, they are in the process of redesigning their platform almost from scratch with an external vendor, which will release publicly in the near future. When that happens we will update the link to the new repo. The archiving of this one is housekeeping for what's to come.

File: products/project-connect.json (hunk 2 above)
Comment (lacabra) [reply products/project-connect.json:8]: Good catch. There is mention of the license in the README, but not the actual LICENSE file. I'm looking into this. Marking it as `blocked` until I figure this out.

Comment (lacabra) [reply products/project-connect.json]: A proper LICENSE file was added to that repo, and the link updated in bbc1f37

**Review by conradsp — APPROVED** [review]

**Pull request merged**